### DISCUSSION TITLE
Hello, I really want to get an internship in FrontEnd

### DISCUSSION THREAD

**Discussion by dkcodec** [community/community#90646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, I want to get an internship in FrontEnd, but I don’t know how to find it or which companies to contact.

Maybe someone has some internships for students?

My stack:
-React redux
-Vanilla JS
-Jest 
-some design stuff like ANTD, MUI...
-algorithms
-and some other base stuff 

pls help or what do you recommend for me?

**Comment by sivakumar-mahalingam** [comment]
You can try [The Forage](https://www.theforage.com/)

**Accept my answer if it is helpful**
